packages/js/src/utils/id-token.test.ts

```diff
@@ -0,0 +1,36 @@
+import { SignJWT, generateKeyPair } from 'jose';
+import { StructError } from 'superstruct';
+
+import { decodeIdToken } from './id-token';
+
+describe('decodeIdToken', () => {
+  test('decoding valid JWT should return claims', async () => {
+    const { privateKey } = await generateKeyPair('RS256');
+    const jwt = await new SignJWT({})
+      .setProtectedHeader({ alg: 'RS256' })
+      .setIssuer('foo')
+      .setSubject('bar')
+      .setAudience('qux')
+      .setExpirationTime('2h')
+      .setIssuedAt()
+      .sign(privateKey);
+    const idTokenClaims = decodeIdToken(jwt);
+    expect(idTokenClaims.sub).toEqual('bar');
```

Comment:
shall we just use equal function for object?

Reply:
Updated, PTAL.